browser/main/SideNav/StorageItem.js

```diff
@@ -131,8 +134,39 @@ class StorageItem extends React.Component {
     }
   }
 
+  handleDragStart (e, folder) {
+    e.dataTransfer.setData('dragFolderKey', folder.key)
+  }
+
+  handleDrop (e, folder, storage, dispatch) {
+    const dragFolderKey = e.dataTransfer.getData('dragFolderKey')
+    this.changeOrderFolder(dragFolderKey, folder.key, storage.folders)
+    CSON.writeFileSync(path.join(storage.path, 'boostnote.json'), _.pick(storage, ['folders', 'version']))
```

Comment:
Thank you commenting, at first I thought that would be nice, but when I actually did that, it did not work properly. But, I think I will try using dataApi again.